docs/source/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+# sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('../../'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Hyperledger Cactus'
+copyright = '2021, Hyperledger Cactus'
+author = 'Hyperledger Cactus'
+
+# The full version, including alpha/beta/rc tags
+release = '4.0.0'
```

Comment:
@AzaharaC If this is the release of Cactus then you could parse the `lerna.json` file's JSON content and retrieve the `version` in there. The file is in the project root and looks like this:

```json
{
  "packages": [
    "packages/cactus-*",
    "examples/cactus-*"
  ],
  "version": "0.4.1",
  "command": {
    "version": {
      "message": "chore(release): publish"
    }
  },
  "changelogPreset": "angular"
}

```

If this release is something else, then please disregard this comment.